### DISCUSSION TITLE
atoti-jupyterlab3 package does not have 0.8.4 version

### DISCUSSION THREAD

**Discussion by amhatre** [atoti/atoti#810] [Q&A] [answered]
Hello,
After installing Atoti 0.8.4, I see that atoti-jupyterlab3 v0.8.3 is installed. Everything is running fine. Just want to check if it is expected, and if there is any reason 0.8.4 was not released for atoti-jupyterlab3?

Atoti package list - 
![image](https://github.com/atoti/atoti/assets/879533/e67e4ef1-fd60-487d-bbfc-d95ac4adf570)

Thanks,
Amit

**Comment by tibdex** [comment, marked as answer]
Hi Amit, this is the same question as this [internal one](https://activeviam.atlassian.net/browse/PYTHON-175) but I'll repeat the answer here so that everyone can see it:

`atoti` >= 0.8.3 is compatible with `atoti-jupyterlab3` 0.8.3. We stopped publishing new versions of `atoti-jupyterlab3` to progress on the upgrade to JupyterLab 4. This upgrade is now done so, in Atoti 0.8.5, you’ll be able to switch to `atoti-jupyterlab` 0.8.5 which uses JupyterLab >= 4.0.5 and Atoti UI 5.1.9.
